### DISCUSSION TITLE
Cancellation request, who all gets the email?

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretix/pretix#4053] [Q&A] [answered]
When a guest requests a cancellation, an email is sent to the organisers with the request. 

Is there an option to disable this in the organiser permissions for individual teams?
Or in other words, which of the options in the organiser permissions settings enable/disable this?

Thanks a lot!

**Comment by pc-coholic** [comment, marked as answer]
The email notification is not tied to an event but rather set for every user in their own profile's notification settings.

From the system side, the notification is only sent to people who have enabled that notification and who have the "can view orders"-permission.
